apps/condo/domains/organization/utils/testSchema/index.js

```diff
@@ -24,21 +25,35 @@ const OrganizationLink = generateGQLTestUtils(OrganizationLinkGQL)
 const TokenSet = generateGQLTestUtils(TokenSetGQL)
 /* AUTOGENERATE MARKER <CONST> */
 
-async function createTestOrganization (client, extraAttrs = {}) {
+async function readOrganization (client, id) {
+    if (!id) throw new Error ('no id')
+
+    return await Organization.getAll(client, { id })
+}
+
+/**
+ * Creates test organization prefilled with faker data, hydrated with extraAttrs and extraMeta, if provided
+ * @param client
+ * @param extraAttrs
+ * @param extraMeta {inn?: string, country?: string}
+ * @returns {Promise<({data: *, errors: *}|*|{country: (*|string), dv: number, sender: {dv: number, fingerprint: *}, meta: {zipCode: *, number: *, country: (*|string), dv: number, city, street: *, inn: (*), kpp: *}, name: *, description: *})[]>}
+ */
+async function createTestOrganization (client, extraAttrs = {}, extraMeta = {}) {
     if (!client) throw new Error ('no client')
+
     const sender = { dv: 1, fingerprint: faker.random.alphaNumeric (8) }
-    const country = DEFAULT_ENGLISH_COUNTRY
+    const country = get(extraMeta, 'country') || DEFAULT_ENGLISH_COUNTRY
```

Comment:
It’s redundant. You can override. Look at

Reply:
removed